pkg/plugins/network/templates/kubeovn.go

```diff
@@ -1184,7 +1185,10 @@ spec:
     kind: HtbQos
     shortNames:
       - htbqos
----
+`)))
+
+	OVN = template.Must(template.New("ovn.yaml").Parse(
+		dedent.Dedent(`---
 {{ if .DpdkMode }}
```

Comment:
I see the DpdkMode has been removed on the above, so is this piece of code still needed?

Reply:
DpdkMode was in this location before the network-plugin.yaml was split

Reply:
OK, I got it.